deploy/helm/cf-operator/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  - name: quarks-job
+    version: 0.0.0+0.g06ad02b
```

Comment:
I'd prefer it, if we had to change the version only in one file. Can we use `$QUARKS_JOB_HELM_VERSION` here?
Also this says 06ad02b and bin/include/dependencies says 491e84a

Reply:
I missed the replace script

Reply:
I used `0.0.0` as placeholder.